### PR TITLE
It 2820 auxtel ccs

### PR DESCRIPTION
**Pull request by glennmorris**
A simple way to allow extra lines to be added to some /etc/ccs property files.
Needed by auxtel.

### CODE REVIEW

File: manifests/config.pp [hunk 1]
```diff
@@ -8,17 +8,19 @@
...
-    'udp_ccs.properties',
-  ]
-
-  $epp_vars = {
-    'hostname' => $trusted['certname'],
+  ## Hash of templates and any arguments they take.
+  $etc_files = {
+    'logging.properties' => {},
+    'ccsGlobal.properties' => {
+      'global_properties' => lookup('ccs_software::global_properties', Array[String], 'unique', []),
```
Comment (jhoblitt) [inline manifests/config.pp:15]: It isn't necessary to do a hiera lookup here as these parameters can be added to the `ccs_software` class and referred to directly.  That parameter approach is also more flexible as the module could be  configured both from puppet code and via hiera automatic binding.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: manifests/config.pp (hunk 1 above)
Comment (glennmorris) [reply manifests/config.pp:15]: Does that do merging? I want the "unique" merge strategy.

Comment (jhoblitt) [reply manifests/config.pp:15]: That policy can go into the hierarchy. E.g.: https://github.com/lsst-it/lsst-itconf/blob/master/hieradata/org/lsst.yaml#L2-L32

Comment (glennmorris) [reply manifests/config.pp]: Thanks for the feedback; see latest version.

**Pull request merged**